rocketpy/motors/motor.py

```diff
@@ -503,9 +501,9 @@ def I_11(self):
         The e_1 direction is assumed to be the direction perpendicular to the
         motor body axis. Also, due to symmetry, I_11 = I_22.
 
-        References
-        ----------
-        .. [1] https://en.wikipedia.org/wiki/Moment_of_inertia#Inertia_tensor
+        See Also
+        --------
+        https://en.wikipedia.org/wiki/Moment_of_inertia#Inertia_tensor
```

Comment:
Any reason to prefering the ``See Also``? I believe the warnings that were raised by the ``References`` section were caused since this ``[1]`` was not being referenced in the docstring text.

Reply:
The problem was actually different: multiple references were being added with the number 1.
Unfortunatelly sphinx shares these numbers within the whole documentation project.

Also, the See Also is a special component that gets redered as a snippet after building the html docs.